widget/ui/src/components/BestRoute/BestRoute.styles.ts

```diff
@@ -162,7 +144,7 @@ export const Separator = styled('div', {
   height: '$12',
   marginLeft: '$10',
   marginRight: '$10',
-  borderLeft: '1px solid $foreground',
+  borderLeft: '0.5px solid $foreground',
```

Comment:
0.5px? really? I think the minimum should be 1px.

Reply:
fixed.